lib/collector/serverless.js

```diff
@@ -35,6 +35,10 @@ class ServerlessCollector {
       agent_version: agent.version
     }
     this.payload = {}
+    this.pipePath = (agent.pipePath || process.env.pipePath) ?
```

Comment:
@astormnewrelic had suggested making the path an injectable constructor property.  I'd prepped for that, but then bumped into what you mention, about the agent not having a pipePath. I'd be open to making it a constructor property...but I'm not quite sure where the value would be defined for that.

Reply:
ah. you could have a constructor variable (makes testing easier) that defaults to the values you have here w/o the agent part.

so...

```js
constructor(agent, pipePath) {
  // ...
  this.pipePath = pipePath || ...
```

could also do param defaulting but guessing taht would be a bit much with the multiple assigned values.

Or, you could inject these values from the agent side. In which case, the grabbing of the pipe path *could* come from the config (either file or env var), if that's something we potentially want... instead of a one-off ENV var or default only. `this.collector = new ServerlessCollector(this, this.config.lambda_pipe_path)` or some such.

This can be done in future PR's (or wait for need) if don't want to do with this. But I don't think we want the unused agent.pipePath in there for this PR.

Reply:
I've removed the unused agent property.  I'm thinking that `lib/agent.js` would be the right place to set that path on the collector--would that be right?